userguide/forms.adoc

```diff
@@ -350,7 +350,7 @@ To peform a range check on a numeric field you can add something like:
 
 [source,json]
 ----
-    "range":"[0,10)"
```

Comment:
HI, this is actually right. The ) bracket stands for exclusive, the ] for inclusive

Reply:
Ah OK, I apologies, I'm not familiar with that notation I just saw it as an" un-balanced" string and assumed it was a typo error. May be the text could be edited to explain this meaning of the different forms of inclusion?